src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -562,16 +562,21 @@ public void onServerTick(@NotNull final TickEvent.ServerTickEvent event)
     /**
      * Update Subscribers with Colony, Citizen, and AbstractBuilding Views.
      */
-    public void updateSubscribers()
+    private void updateSubscribers()
     {
+        // If the world or server is null, don't try to update the subscribers this tick.
+        if (world == null || world.getMinecraftServer() == null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)